packages/adapters/wagmi/src/client.ts

```diff
@@ -230,10 +230,6 @@ export class WagmiAdapter implements ChainAdapter {
 
     this.networkControllerClient = {
       switchCaipNetwork: async caipNetwork => {
-        SafeLocalStorage.setItem(
-          SafeLocalStorageKeys.ACTIVE_CAIP_NETWORK,
-          JSON.stringify(caipNetwork)
-        )
```

Comment:
is this intended if so, where are we persisting the netowkr?

Reply:
We are persisting the network on where we update the state, we shouldn't be calling it from different places. 

Like if you do;
```js
// pseudo code
ChainController.state.activeCaipNetwork = something
```

You can update localStroge here as well
```js
// pseudo code
ChainController.state.activeCaipNetwork = something
localStorage.setItem("activeCaipNetwork, something) 
```